README.md

```diff
@@ -365,7 +329,11 @@ spec:
   image: "quay.io/kiegroup/kogito-data-index:latest"
   kafka:
     # the service name and port for the kafka cluster. Example: my-kafka-cluster:9092
-    serviceURI: my-cluster-kafka-bootstrap:9092
+    externalURI: my-cluster-kafka-bootstrap:9092
```

Comment:
Why did we change to externalURI ?
Infinispan is still serviceURI ... We need some coherence here ;)

Reply:
I'll change the Infinispan `Spec` on this JIRA: https://issues.jboss.org/browse/KOGITO-165
I told him to use `ExternalURI` since makes more sense now that we will use `service` for internal infra.

Reply:
ok good for me then ;)

Reply:
Actually I'll open another JIRA for this